crates/vizia_core/src/style/mod.rs

```diff
@@ -95,6 +96,19 @@ pub struct Style {
     pub disabled: StyleSet<bool>,
     pub abilities: SparseSet<Abilities>,
 
+    // Accessibility properties
+    pub roles: SparseSet<Role>,
+    pub default_action_verb: SparseSet<DefaultActionVerb>,
+    pub live: SparseSet<Live>,
+    pub labelled_by: SparseSet<Entity>,
+    pub hidden: SparseSet<bool>,
+    // TODO: Should we store these or somehow derive them from app state?
```

Comment:
But then we would still need to store this state somewhere so that we can build the accesskit node. Note that entities which don't have these properties don't have a space in the sparse set.